docs/data/material/components/cards/RecipeReviewCard.js

```diff
@@ -18,12 +18,25 @@ import MoreVertIcon from '@mui/icons-material/MoreVert';
 const ExpandMore = styled((props) => {
   const { expand, ...other } = props;
   return <IconButton {...other} />;
-})(({ theme, expand }) => ({
-  transform: !expand ? 'rotate(0deg)' : 'rotate(180deg)',
+})(({ theme }) => ({
   marginLeft: 'auto',
   transition: theme.transitions.create('transform', {
     duration: theme.transitions.duration.shortest,
   }),
+  variants: [
+    {
+      props: ({ expand }) => !expand,
+      style: {
+        transform: 'rotate(0deg)',
+      },
+    },
+    {
+      props: ({ expand }) => !!expand,
```

Comment:
This is funny :D, we can manually change it

Reply:
Yeah, this kind of transformation has to be done manually. The codemod cannot decide on this because there are cases where the variable is not boolean.

Reply:
Yeah, I understand that, the output of the codemod makes sense based on the previous code